components/gantry/oneaxis/oneaxis.go

```diff
@@ -299,8 +298,9 @@ func (g *oneAxis) testLimit(ctx context.Context, zero bool) (float64, error) {
 			return 0, ctx.Err()
 		}
 	}
-
-	return g.motor.Position(ctx, nil)
+	position, err := g.motor.Position(ctx, nil)
+	time.Sleep(1 * time.Second)
```

Comment:
Surely this line isn't supposed to be here, right? I'd undo the changes to this paragraph.

Reply:
The precision with the pause and the crappy limit switches makes perfect sense. However, can we pause for less time? Also, make the time delay a constant at the top of the file. also, we have to make sure that `testLimit` is only called in the homing sequence now, I don't want a pause interfering with a Movement.

Reply:
yeah I can do that. the homing sequence is the only time I'm calling `testLimit` so that should be fine

Reply:
From the code itself, I had no clue that this was about improving the precision. Put in a comment explaining what's going on here.